maxima/g0/moments/gkIntMomentsFuncs.mac

```diff
@@ -9,6 +9,130 @@ fpprec : 24$
 
 volExpr(totDim) := prod(dxv[i-1], i, 1, totDim)$
 
+
+/* Integrated M0. */
+calcGkIntM0(fh, funcNm, cdim, vdim, basisFun, polyOrder) := block(
+  [varsC,bC,varsP,bP,vSub,vmap_e,vmapSq_e,vmap_prime_e,vmap_c,
+   d,f_e,M,m0,m1,m2,bmag_e,Mtemp,tmp_e,tempPowVars],
```

Comment:
these lists have to include the temporary variables in this function. I think some of these variables don't actually exist in these functions.